tests/fixtures/validators.yml

```diff
@@ -27,4 +27,70 @@ silent_mutation:
     - query: NP_000542.1:p.Pro61=
     - query: NP_000918.2:p.Ile1145=
   should_not_match:
-    - query: NP_000542.1:p.Pro62=
\ No newline at end of file
+    - query: NP_000542.1:p.Pro62=
+
+coding_dna_substitution:
+  should_match:
+    - query: NM_004333.4:c.1799T>A
+    - query: ENST00000288602.6:c.1799T>A
+    - query: BRAF (c.1799T>A)
+    - query: BRAF c.1799T>A
+    - query: BRAF V600E c.1799T>A
+    - query: LRG_199t1:c.54G>H
+    - query: LRG_199p1:c.54G>H
+    - query: LRG_199:c.54G>H
+  should_not_match:
+    - query: BRAF c.18000000000000T>A
+    - query: NM_004333.4:c.17699T>A
+
+genomic_substitution:
+  should_match:
+    - query: NC_000007.13:g.140453136A>T
+    - query: NC_000007.13:g.55259515T>G
+  should_not_match:
+    - query: NC_000007.13:g.1436A>T
+    - query: NC_000007.13:g.4T>A
+
+coding_dna_silent_mutation:
+  should_match:
+    - query: NM_004006.2:c.123=
+    - query: NM_004333.4:c.1799=
+    - query: ENST00000288602.6:c.1799=
+    - query: BRAF c.1799=
+    - query: BRAF V600E c.1799=
+  should_not_match:
+    - query: NM_004006.2:c.13994=
+    - query: BRAF c.18000000000000=
+
+genomic_silent_mutation:
+  should_match:
+    - query: NC_000007.13:g.140453136=
+    - query: NC_000007.13:g.55259515=
+  should_not_match:
+    - query: NC_000007.13:g.159138664=
+
+coding_dna_delins:
+  should_match:
+    - query: NM_001289937.1:c.2326_2327delinsCT
+    - query: NM_000551.3:c.615delinsAA
+    - query: ENST00000440973.1:c.1607_1608delinsAG
+      # TODO: Find this gene symbol
+#    - query: ENST00000277541.6:c.7330delinsACA
```

Comment:
should this have a separate issue/reminder to return to?

Reply:
Yes, I'll add it.